octoprint_prometheus_exporter/gcodeparser.py

```diff
@@ -90,19 +93,31 @@ def parse_fan_speed(self, line):
 
         return None
 
+    def process_axis_movement(self, target_position, current_position):
```

Comment:
Don't we have abs? like `abs(target-currennt)`

Reply:
Good catch, testing that.